### PR TITLE
Update bakery to work with separated git resources

### PR DESCRIPTION
**Pull request by rnathuji**

### CODE REVIEW

File: bakery/src/scripts/fetch_map_resources.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+def main():
+    in_dir = Path(sys.argv[1]).resolve(strict=True)
+    resource_rel_path_prefix = sys.argv[2]
+
+    cnxml_files = in_dir.glob("**/*.cnxml")
+
+    for cnxml_file in cnxml_files:
+        doc = etree.parse(str(cnxml_file))
+        for node in doc.xpath(
+            '//x:image',
```
[the remark this answers is not in the document]
Comment (tomjw64) [reply bakery/src/scripts/fetch_map_resources.py:17]: evidence of this pending :laughing:

[the remark this answers is not in the document]
Comment (rnathuji) [reply bakery/src/scripts/fetch_map_resources.py:17]: Drawing on the convo from our Slack thread: It seems the hrefs to images in the XHTML are being populated by `cnx-epub` based upon the model built by `neb` during assemble which in turn relies upon the existence of the resource files it [finds in a module](https://github.com/openstax/nebuchadnezzar/blob/f7dedd998d4931b68c70c188d0be39fc5a5427ed/nebu/models/document.py#L143-L147).

**Comment by rnathuji** [comment]
@tomjw64 - I believe we addressed your [comment](https://github.com/openstax/output-producer-service/pull/221#discussion_r520753788) on the `fetch_map_resources.py` script in that all images should be covered by looking for `<image>`. Were there any other comments on this PR to address?

**Review by tomjw64 — APPROVED** [review]

**Comment by rnathuji** [comment]
Just a note on this PR: I'm holding off merging until [this PR](https://github.com/openstax/ce-git-storage-spike/pull/4) is approved / ready to merge to avoid breaking the pipeline for folks still pointing to `master` / `latest` when testing the git pipeline.

**Pull request merged**